sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/Generated/Models/SKU.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.DeviceUpdate.Models
+{
+    /// <summary> Device Update Sku. </summary>
+    public readonly partial struct SKU : IEquatable<SKU>
```

Comment:
Could we also rename this one? According to the .NET convention, we should use pascal case for acronyms over two letters. I think you can rename it to `Sku` and our generator will automatically change the name to `DeviceUpdateSku`.

Reply:
This was picked up from swagger https://github.com/Azure/azure-rest-api-specs/blob/c2d25c0f023edd1fea753a89d6409f3e7a9f8956/specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2022-04-01-preview/deviceupdate.json#L1335

              "x-ms-enum": {
                "name": "SKU",
                "modelAsString": true
              }

Reply:
Same as above.